### DISCUSSION TITLE
Multiple Issues ( AMS, F123, Dirt 4)

### DISCUSSION THREAD

**Discussion by Zordinator** [gotzl/hid-fanatecff#79] [Q&A] [answered]
I opening this here, as i don't know if its an actual issue with a driver or if there is an issue on my side. I have installed the driver just like it was instructed in the readme, and it at least worked with AMS 2 with Proton expreimental in the beginning woth my DD Pro GT. The next day i had no FFB in AMS2, but the buttons on the Wheel and turning and the Pedals are still working, i then reinstalled the driver, but with no luck the issue is still there. I then tried F1 23, and under Proton 9, but that wont even start when the wheel is connected. If you start it without the Wheel turned on it opens and works, but the moment i turn the wheel on the Game crashes. I then tried Dirt 4 as it is native. and the wheel isn't even recognized there.
Also the directory /sys/module/hid_fanatec/  to get to the advanced settings  is missing, that might be a clue.
I use Linux Mint 21.3 and the Fanatec Firmware on the DD Pro Base and Wheels are all up to date.

**Comment by SynthetikzZ** [comment, marked as answer]
After an Linux Kernel update you always have to rebuild and reinstall the driver. CD into the driver folder and issue a make and make install again. Also you have to rebuild your initramfs with dracut to make the driver persistent after reboot of the PC.

**Comment by gotzl** [comment]
> Also the directory /sys/module/hid_fanatec/ to get to the advanced settings is missing, that might be a clue.

Jep, this means that the driver is not loaded, probably because it is not installed correctly for whatever reason ...
Note that after reinstalling the driver, you need to power-cycle (or plug-out/plugin-in) the wheel for the driver to get loaded.

> but the moment i turn the wheel on the Game crashes

This sounds like #67, please try the workaround mentioned in the README.

**Comment by Zordinator** [comment]
ok looks like i found the Problem. Before, when i tried to reinstall the driver,  I rebuilt the driver in the folder, where I issued the make and make install the first time already. but after extracting the ZIP file again into another directory, and issuing a make and make install there, it actually worked, and now the FFB is back for AMS2 and also the Workaround with Proton 6 worked for F1 23. 
Also thanks for the Tip from SyntheikzZ with the initramfs. I added the module into it with dracut, and now it still works after a reboot.
